apps/system/js/statusbar.js

```diff
@@ -400,7 +404,8 @@ var StatusBar = {
     }
 
     this.element.classList.toggle('maximized',
-      app.isHomescreen || !!(app.appChrome && app.appChrome.isMaximized()));
+      app.isHomescreen || app.isAttentionWindow ||
```

Comment:
What if we make sure attentWindow.appChrome.isMaximized() === true and we don't need any special check here?